ale_linters/less/lessc.vim

```diff
@@ -0,0 +1,36 @@
+" Author: zanona <https://github.com/zanona>
+" Description: This file adds support for checking Less code with lessc.
+
+call ale#Set('less_lessc_options', '')
+
+function! ale_linters#less#lessc#GetCommand(buffer) abort
+    return 'cat %t | lessc'
```

Comment:
Remove `'cat %t |'` here. That won't work on Windows. If you don't put `%t` in the command, ALE will automatically put `< temporary_file` on the end for you.